sdk/eventgrid/Azure.Messaging.EventGrid/CHANGELOG.md

```diff
@@ -1,7 +1,6 @@
 # Release History
 ## 4.0.0-beta.3 (Unreleased)
 
```

Comment:
Looks unrelated.

Reply:
I fixed that spacing as part of a rebase which is why it showed up.